app/src/main/java/gov/hhs/cdc/trustedintermediary/auth/AuthRequestValidator.java

```diff
@@ -21,10 +21,12 @@ public class AuthRequestValidator {
     private static final AuthRequestValidator INSTANCE = new AuthRequestValidator();
 
     @Inject private AuthEngine jwtEngine;
-    @Inject private Cache keyCache;
+    @Inject Cache keyCache;
     @Inject private Secrets secrets;
     @Inject private Logger logger;
 
+    String ourPublicKey = "trusted-intermediary-public-key-" + ApplicationContext.getEnvironment();
```

Comment:
Do we want this public key name hardcoded here? And if so do we need a #pragma allow secret

Reply:
It previously existed, but I moved it to a higher level so it's accessible in tests. I can add the #pragma allow secrets if we want but I also don't think it got flagged before as a secret.

Reply:
Cool